azext_iot/central/params.py

```diff
@@ -386,3 +384,80 @@ def load_central_arguments(self, _):
             options_list=["--description", "--desc"],
             help="Detailed description of the job.",
         )
+
+    with self.argument_context("iot central query") as context:
+        context.argument(
+            "query_string",
+            options_list=["--query-string", "--qs"],
+            help="Query clause to retrieve telemetry or property data.",
+        )
+        context.argument(
+            "api_version",
+            options_list=["--api-version", "--av"],
+            choices=CaseInsensitiveList([ApiVersion.v1_1_preview.value]),
+            default=ApiVersion.v1_1_preview.value,
+            help="The API version for the requested operation.",
+        )
+
+    with self.argument_context("iot central export destination") as context:
+        context.argument(
+            "destination_id",
+            options_list=["--dest-id"],
+            help="Unique identifier for the destination.",
+        )
+        context.argument(
+            "api_version",
+            options_list=["--api-version", "--av"],
+            choices=CaseInsensitiveList([ApiVersion.v1_1_preview.value]),
+            default=ApiVersion.v1_1_preview.value,
+            help="The API version for the requested operation.",
+        )
+
+    with self.argument_context("iot central export destination create") as context:
+        context.argument(
+            "content",
+            options_list=["--content", "-k"],
+            help="The destination definition. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example:./path/to/file.json]"
+            " [Example of stringified JSON:{<Destination Data JSON>}]. The request body must contain content of Destination.",
+        )
+    with self.argument_context("iot central export destination update") as context:
+        context.argument(
+            "content",
+            options_list=["--content", "-k"],
+            help="The partial destination definition. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example:./path/to/file.json]"
+            " [Example of stringified JSON:{<Destination Data JSON>}."
```

Comment:
Do you think these json examples are effective enough for someone to understand how to use the commands? Should we expand with more realistic examples, or provide a link to a source with those examples? Also this line is missing a closing square bracket `]`.

Reply:
Here should be enough, I added a list of real examples from help file for both destination & export, should be able to give user more details how to construct the payload for different destination.

Reply:
Excellent.